db/migrate/20200224180456_add_custom_user_columns.rb

```diff
@@ -0,0 +1,14 @@
+class AddCustomUserColumns < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :first_name, :string
+    add_column :users, :last_name, :string
+    add_column :users, :phone_number, :string
+    add_column :users, :age, :integer
+    add_column :users, :gender, :string
+    add_column :users, :language, :string
+    add_column :users, :location, :string
+    add_column :users, :qualifications, :string
```

Comment:
Hi Janette, qualifications doesn't need to be a column in this table since we have a separate table for that.

Reply:
Done